native/packages/react-native-bpk-component-text-input/src/BpkTextInput.js

```diff
@@ -24,8 +26,54 @@ import { animationDurationSm } from 'bpk-tokens/tokens/base.react.native';
 import { ValidIcon, InvalidIcon } from './BpkTextInputIcons';
 import { getLabelStyle, getInputContainerStyle, styles } from './styles';
 
-class BpkTextInput extends Component {
-  constructor(props) {
+type Props = {
+  label: string,
+  value: string,
+  clearButtonMode: 'never' | 'while-editing' | 'unless-editing' | 'always',
+  editable: boolean,
+  inputRef: ?() => void,
```

Comment:
This is incorrect. It should be taking an argument with the ref(no idea what type that should be)

Reply:
Have updated